src/dot_recognize/src/point_number_publisher.py

```diff
@@ -135,10 +135,10 @@ def numbercallback(self,data):
 
 if __name__ == '__main__':
     bridge=CvBridge()
-    cap =cv2.VideoCapture(2)
+    cap =cv2.VideoCapture(0)
     bridge=CvBridge()
     ret, frame=cap.read()
```

Comment:
Can you add a feature that checks whether the camera read successfully? This will be helpful. A basic `if` with the boolean value `ret` will do the job.